addons/website_sale_autocomplete/static/src/js/address_form.js

```diff
@@ -38,30 +38,32 @@ publicWidget.registry.AddressForm = publicWidget.Widget.extend({
         });
     },
 
-    _onChangeStreet: async function(ev) {
-        this.dp.add(
-            this._rpc({
-                route: '/autocomplete/address',
-                params: {
-                    partial_address: ev.currentTarget.value,
-                    session_id: this.sessionId || null
-                }
-            })).then((response) => {
-               if (response.results) {
-                   const inputContainer = ev.currentTarget.parentNode;
-                   const dropdown = inputContainer.querySelector('.dropdown-menu');
-                   if (dropdown) {
-                       dropdown.remove();
+    _onChangeStreet: async function (ev) {
+        if (ev.currentTarget.value.length >= 5) {
```

Comment:
I don't have a key to test right now but I wonder if we shouldn't remove the dropdown in case the input is less than 5 characters so that we don't just have a hanging dropdown that may or may not be related to the current input.

Reply:
yes, I already removed the dropdown as soon you don't have result locally.